README.md

```diff
@@ -184,6 +209,17 @@ tox -e uats-remote -- --include-gpu-tests --filter "kfp"
 
 As shown in the example above, tests under the `gpu` directory follow the same filters with the rest of the tests.
 
+##### Taints and tolerations
+
+If a [taint](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) is used to prevent scheduling unintended workload to GPU nodes, a toleration is needed in order to enable GPU tests to schedule workloads. This is achieved via the `--toleration` argument which enables passing the sub-arguments `key, operator, value, effect, seconds`. For example:
+
+```bash
+tox -e uats-remote -- --include-gpu-tests --toleration key="MyKey" value="gpu" effect="NoSchedule"
```

Comment:
We should change this example to specific example with the taint and concrete toleration example.

We should add info that this toleration is added as a poddefault to cluster which will be bind to pods with label `enable-gpu` `true`